sdk/go/readme.md

```diff
@@ -1,13 +1,13 @@
 # The (Tiny)Go SDK for Spin
 
 This package contains an SDK that facilitates building Spin components in
-(Tiny)Go. It currently allows building HTTP components that target the Wagi
+(Tiny)Go. It allows building HTTP components that target the Spin
 executor.
 
 ```go
 func main() {
- // call the HandleRequest function
- spin_http.HandleRequest(func(w http.ResponseWriter, r *http.Request) {
+ // call the Handle function
+ spin_http.Handle(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This example should be similar to https://github.com/fermyon/spin/blob/main/examples/http-tinygo/main.go.

Reply:
Indeed, `main()` doesn't work now. `init()` is needed instead.